antiplag/models.py

```diff
@@ -12,6 +12,7 @@ class SubmissionStatus(models.TextChoices):
     status = models.CharField(max_length=10, choices=SubmissionStatus.choices)
     created_at = models.DateTimeField(auto_now_add=True)
     updated_at = models.DateTimeField(auto_now=True)
+    email = models.EmailField(max_length=254, null=True)
```

Comment:
is the `max_length` bases on email standard? If so, maybe move it to `constants.py`.

Reply:
Yeah, I'd leave it here, I don't think it will be used anywhere else.

Reply:
Well next time somebody looks over this code, he will be wondering why `254` and if it has any meaning. If you move it to `constants.py`, then it will be clear it's the `EMAIL_ADDRESS_RFC5321_LENGTH` and not some arbitrary length.